lib/grpc/GrpcService.ts

```diff
@@ -431,6 +432,32 @@ class GrpcService {
     }
   }
 
+  /**
+   * See [[Service.listTrades]]
+   */
+  public listTrades: grpc.handleUnaryCall<xudrpc.ListTradesRequest, xudrpc.ListTradesResponse> = async (call, callback) => {
+    try {
+      const listTrades = await this.service.listTrades(call.request.toObject());
```

Comment:
This can just be `trades`. Calling it `listTrades` makes it sounds like a function rather than a collection of trades.